homeassistant/components/ecovacs/quality_scale.yaml

```diff
@@ -0,0 +1,95 @@
+rules:
```

Comment:
Why don't you make the event name the image entity map name?

Reply:
Discussed on Discord, not needed